### DISCUSSION TITLE
I'd like to use source code to deploy ChatGPT-Next-Web in my ubuntu 22.04. Anyone can help to show me a guide? thanks.

### DISCUSSION THREAD

**Discussion by yankevin** [ChatGPTNextWeb/NextChat#3276] [General]
sudo apt install yarn nodejs npm.  -- i do the install command in my ubuntu 22.04 sucessfully. 

but, what are the next steps should I do. thanks.

**Comment by H0llyW00dzZ** [comment]
yarn run dev, or use docker 
commands : 
```sh
docker pull yidadaa/chatgpt-next-web
```

other commands : https://github.com/Yidadaa/ChatGPT-Next-Web?tab=readme-ov-file#docker-recommended
